code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -180,6 +180,22 @@
 	max_ammo = 8
 	multiple_sprites = 2
 
+/obj/item/ammo_box/magazine/m10mm/fire
+	name = "pistol magazine (10mm incendiary)"
+	desc = "A gun magazine. Loaded with rounds which ignite the target."
+	ammo_type = /obj/item/ammo_casing/c10mm/fire
```

Comment:
All these types are not defined. Not sure why you removed them; some of their code needed to be removed, but not their underlying types.